docs/howto/custom-template-tags.txt

```diff
@@ -459,6 +459,18 @@ positional arguments. For example:
 
     {% my_tag 123 "abcd" book.title warning=message|lower profile=user.profile %}
 
+.. versionadded:: 1.8
+
+It's possible to store the tag results in a template variable rather than
+directly outputting it. This is done by using the ``as`` argument followed by
+the variable name. Doing so enables you to output the content yourself where
+you see fit:
```

Comment:
Isn't the idea more avoiding repeated calls to the same template tag if it's used multiple times?